source/ContentScript/recorder.ts

```diff
@@ -351,6 +354,68 @@ class Recorder {
       isDragging = false;
     });
   }
+
+  async notify(stmt: ZestStatement): Promise<void> {
+    const notifyMessage = {
+      title: '',
+      message: '',
+    };
+
+    if (stmt instanceof ZestStatementElementClick) {
+      notifyMessage.title = 'Click';
+      notifyMessage.message = stmt.elementLocator.element;
+    } else if (stmt instanceof ZestStatementElementSendKeys) {
+      notifyMessage.title = 'Send Keys';
+      notifyMessage.message = `${stmt.elementLocator.element}: ${stmt.keys}`;
+    } else if (stmt instanceof ZestStatementLaunchBrowser) {
+      notifyMessage.title = 'Launch Browser';
+      notifyMessage.message = stmt.browserType;
+    } else if (stmt instanceof ZestStatementSwitchToFrame) {
+      notifyMessage.title = 'Switch To Frame';
+      notifyMessage.message = stmt.frameIndex.toString();
+    }
+
+    // wait for previous notification to be removed
+    if (this.isNotificationRaised) {
+      await this.waitForNotificationToClear();
+    }
+    const floatingDiv = document.getElementById('ZapfloatingDiv');
+    if (!floatingDiv) {
+      console.log('Floating Div Not Found !');
+      return;
+    }
+
+    this.isNotificationRaised = true;
+    const messageElement = document.createElement('p');
```

Comment:
Should keep using the same `className` to avoid recording events when the message is showing?

Reply:
yess

Reply:
fixed